front/app/component-library/components/Checkbox/index.tsx

```diff
@@ -112,6 +115,7 @@ const Checkbox = ({
   onChange,
   checked,
   checkedColor,
+  usePrimaryBorder,
```

Comment:
```suggestion
  usePrimaryBorder = false,
```
In case we don't want to use `primary` everywhere.